### DISCUSSION TITLE
Vite.js render problem

### DISCUSSION THREAD

**Discussion by ceyhununlu** [quasarframework/quasar#14934] [Quasar App Extensions]
Hi, i created a common library. I used quasar-app-extension-my-component in my project. If I use only html elements, there is no problem. But if I use quasar component in my component, there is problem. It works without any problem when I run my project with quasar dev. But when I do quasar build and quasar serve to my project, related component disappears. my project runs with quasar/vite.js. What should I do to render external quasar components during build?

Here is my component and package json:
<img width="813" alt="Screenshot 2022-11-18 at 22 01 59" src="https://user-images.githubusercontent.com/3489808/202783358-d2c53730-0be5-41f4-9c21-68bba7dad2f6.png">
<img width="626" alt="Screenshot 2022-11-18 at 22 02 33" src="https://user-images.githubusercontent.com/3489808/202783385-e93af4e8-33c8-4d88-8241-22f0e70d6bbb.png">

**Comment by andrejs82git** [comment]
Hi!
I have similar problem.
Pay attention if you setup in quasar.config.js that:
```
viteConf.optimizeDeps = {
         disabled: true
    }
```
the dev server start take similar errors.

**Reply by andrejs82git** [reply]
My solution is:
add to quasar.config.js that code
```
...
      extendViteConf(viteConf) {
        viteConf.resolve.dedupe = ['vue']
        ...
     }
```
